docs/building.md

```diff
@@ -39,6 +39,45 @@ You can use `ligo` to also generate a contract initial storage, using the
 Following are the intructions to generate the initial storages of the DAO configuration
 provided in this codebase, see the [README](../README.md).
```

Comment:
This seems off as there are no instructions "following" (they are after the `Generating a contract metadata` section now).

I suggest moving the new `Generating a contract metadata` section above this `Generating a contract storage` one instead.

Reply:
Make sense, I've updated this.